src/components/Navigation.tsx

```diff
@@ -44,7 +44,7 @@ const Navigation: FC = () => {
       <header className="l-navigation-bar">
         <div className="p-panel is-dark">
           <div className="p-panel__header">
-            <Logo />
+            <Logo disableLink />
```

Comment:
This was a solution to the Wave alert for redundant links since we have two identical links for mobile and web view respectively. I did some more reading on this issue, I don't think it's a WCAG failure but just a Wave alert and it should be acceptable as this is part of the design. I think I will revert this change.